api.go

```diff
@@ -364,40 +366,46 @@ func (api *API) addResource(prototype jsonapi.MarshalIdentifier, source CRUD) *r
 		}
 	}
 
-	api.router.Handle("POST", baseURL, func(w http.ResponseWriter, r *http.Request, params map[string]string) {
-		info := requestInfo(r, api)
-		c := api.contextPool.Get().(APIContexter)
-		c.Reset()
-		api.middlewareChain(c, w, r)
-		err := res.handleCreate(c, w, r, info.prefix, *info)
-		api.contextPool.Put(c)
-		if err != nil {
-			api.handleError(err, w, r)
-		}
-	})
+	if _, ok := source.(CRUDCreate); ok {
```

Comment:
I knew that naming would be a thing here. I wasn't much creative and just tried to keep a reference to the original interface. Your naming makes sense: it describes an action applied to a resource.... committing an update right now ;)